docs/fundamentals/connection/connect-to-mongodb.txt

```diff
@@ -0,0 +1,300 @@
+
+Parts of a Connection URI
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following figure explains each part of a sample connection URI:
+
+.. figure:: /includes/figures/connection_uri_parts.png
+   :alt: Parts of a connection URI
+
+In this connection URI, ``mongodb+srv`` is the protocol, which uses the
+:manual:`DNS Seed List Connection Format </reference/connection-string/#dns-seed-list-connection-format>`
+for greater flexibility in your deployment and the ability to change the
+servers in rotation without reconfiguring clients.
+
+If the machine that hosts your MongoDB deployment does not support this
+feature, use protocol for the
+:manual:`Standard Connection String Format </reference/connection-string/#std-label-connections-standard-connection-string-format>`
+instead.
+
+If you use a password-based authentication, the part of the connection
+string after the protocol contains your username and password. Replace the
+placeholder for ``user`` with your username and ``pass`` with your password.
+If you use an authentication mechanism that does not require a username
+and password, omit this part of the connection URI.
+
+The part of the connection string after the credentials specifies your MongoDB
+instance's hostname or IP address and port. The preceding example uses
+``sample.host`` as the hostname and ``27017`` as the port. Replace these values
+to point to your MongoDB instance.
+
+The last part of the connection string specifies connection and authentication
+options. In the example, we set the following connection options and values:
+
+- ``maxPoolSize=20``
+- ``w=majority``
+
+.. _laravel-database-config:
+
+Laravel Database Connection Configuration
+------------------------------------------
+
+{+odm-short+} lets you configure your MongoDB database connection in the
+``config/database.php`` Laravel application file. You can specify the following
+connection details in this file:
+
+- ``default``, which specifies the default database connection to use when
+  unspecified
+- ``connections``, which contains database connection information to access
+  various databases from your application
+
+You can use the following code in the configuration file to set the default
+connection to a corresponding ``"mongodb"`` entry in the ``connections`` array:
+
+.. code-block:: php
+
+   'default' => 'mongodb',
+
+For a MongoDB database connection, you can specify the following details:
+
+- ``driver``, which specifies the database driver to use for the connection
+- ``dsn``, the data source name (DSN) that specifies the MongoDB connection URI
+- ``host``, which you can use instead of the ``dsn`` setting to specify the
+  network address of one or more MongoDB nodes
+- ``database``, which specifies the name of the MongoDB database to read and
+  write to
+- ``username`` and ``password``, which you can optionally include to specify
+  your database user's credentials to authenticate with MongoDB
```

Comment:
I: right now, it's not very clear that using different properties such as host username password port, etc. is an alternative to including all of this information in the DSN

S: add an admonition or revise these descriptions to make this clear

Reply:
Good idea!  Will add an admonition to explain.